zebra-chain/src/sapling/arbitrary.rs

```diff
@@ -125,7 +125,10 @@ impl Arbitrary for tree::Root {
         (vec(any::<u8>(), 64))
             .prop_map(|bytes| {
                 let bytes = bytes.try_into().expect("vec is the correct length");
-                jubjub::Fq::from_bytes_wide(&bytes).to_bytes().into()
+                jubjub::Fq::from_bytes_wide(&bytes)
+                    .to_bytes()
+                    .try_into()
+                    .expect("bytes are always valid")
```

Comment:
You need a `jubjub::Base`; `jubjub::Scalar` is slightly different. And with that there. will be no error checking needed:

```suggestion
                tree::Root(jubjub::Base::from_bytes_wide(&bytes))
```

Reply:
Ok, this is good, i used it at https://github.com/ZcashFoundation/zebra/pull/3544/commits/ac4a75e7dfd6c1d008b8fe2eb80f8fabd4c54e9d. Thanks.